todo_app/app.py

```diff
@@ -1,14 +1,20 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
+from todo_app.data.session_items import get_items, add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['POST', 'GET'])
 def index():
-    return 'Hello World!'
+    if request.method == 'POST':
+        item_title = request.form['Title']
+        add_item(item_title)
+        return redirect('/')
+    else:
+        items_list = get_items()
+        return render_template('index.html', items_list=items_list)
```

Comment:
These two blocks are totally separate. They should therefore be two separate routes/functions (even if you wanted to keep the path '/' for both of them)